src/main/java/org/opentripplanner/graph_builder/GraphBuilderDataSources.java

```diff
@@ -181,6 +181,10 @@ public NetexFeedParameters getNetexConfig(DataSource dataSource) {
       .orElse(buildConfig.netexDefaults.copyOf().withSource(dataSource.uri()).build());
   }
 
+  public File baseDirectory() {
+    return baseDirectory;
+  }
+
```

Comment:
This is not allowed - the point of the datasource is that it hides where the physical file comes from - Disk, Http, GoogleCloude. I recomend injecting the data source into  StopConsolidationModule and read the data from the Stream. Delete this method.

Reply:
Are you aware that this is not an instance of `DataSource` but rather a the class that holds information about all the data sources?

During a previous dev meeting we agreed that we don't want to use the scanning mechanism but configure the name of the file in the config and resolve it "manually".

Can you advise how this should happen?